src/main/java/org/springframework/data/elasticsearch/core/ReactiveSearchOperations.java

```diff
@@ -206,4 +209,13 @@
 	 * @since 4.0
 	 */
 	Flux<Aggregation> aggregate(Query query, Class<?> entityType, IndexCoordinates index);
+
```

Comment:
Please add another method `Flux<Suggest> suggest(SuggestBuilder suggestion, Class<?> clazz);` where the index name to use is taken from the entity class _clazz_. Check the other methods how the `IndexCoordinates` are set in this case. We need this, as the index name in a `@Document` annotation may contain a SpEL expression that is only resolved when the entity class is used.

Reply:
Added and amended.